zeppelin-web/src/app/notebook/notebook.controller.js

```diff
@@ -398,6 +398,7 @@ function NotebookCtrl($scope, $route, $routeParams, $location, $rootScope,
     } else {
       $scope.note.config.cronExecutingUser = '';
     }
+    $scope.saveNote();
```

Comment:
I think it's a enhancement for timer saver, as we know, if timer saver works well, the paragraph will be saved in 10 seconds， There is a chance that the edited content will lost if it isn't saved in time after GET_NOTE Operation caused. the cron schedule setting cause the GET_NOTE operation, so I think note should be saved in this place.